catalog/ui/search-ui/standard/src/main/webapp/js/view/MetacardDetail.view.js

```diff
@@ -102,9 +106,35 @@ define([
                     $('#nextRecord', this.$el).addClass('disabled');
                 }
             },
+            getNearby: function () {
+                if (!nearbyChecked) {
+                    $('#nearby').html("");
+                    var url = '/services/REST/v1/Locations/nearby/cities';
+
+                    var geometry = this.model.get('geometry');
+                    var wkt = new Wkt.Wkt();
+                    var geoWkt = wkt.read(JSON.stringify(geometry));
+
+                    $.ajax({
+                        url: url,
+                        data: "jsonp=jsonp&wkt="+geoWkt.write(),
```

Comment:
This isn't the right way to do jsonp with jquery. There is a jsonp field that you can attach the callback function to. See https://learn.jquery.com/ajax/working-with-jsonp/